pkg/services/alerting/engine.go

```diff
@@ -57,7 +57,7 @@ type AlertEngine struct {
 
 // IsDisabled returns true if the alerting service is disabled for this instance.
 func (e *AlertEngine) IsDisabled() bool {
-	return e.Cfg.AlertingEnabled == nil || !*(e.Cfg.AlertingEnabled) || !e.Cfg.ExecuteAlerts || e.Cfg.UnifiedAlerting.IsEnabled()
+	return true
```

Comment:
Do we need to follow-up in this PR to remove all calls to `AlertEngine.isDisabled()` or future work?